integrasjonstest/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/integrasjonstest/utils/EndToEndTest.kt

```diff
@@ -106,36 +107,47 @@ val maxMekker =
 abstract class EndToEndTest : ContainerTest() {
     private val imTestRapid = ImTestRapid()
 
+    // Vent på inntektsmeldingdatabase
     private val inntektsmeldingDatabase by lazy {
         println("Database jdbcUrl for im-db: ${postgresContainerOne.jdbcUrl}")
-        postgresContainerOne
-            .toHikariConfig()
-            .let(::Database)
-            .also {
-                val migrationLocation = Path("../db/src/main/resources/db/migration").absolutePathString()
-                it.migrate(migrationLocation)
-            }.createTruncateFunction()
+
+        return@lazy medFlereForsoek(
```

Comment:
Trenger du `return@lazy` her? Som siste linje i lambdaen, så trodde jeg ikke det var nødvendig.

Reply:
Intellij er ikke så begeistret for å ta vekk den, dessverre 🙅 

<img src="https://media1.giphy.com/media/JliGmPEIgzGLe/giphy.gif"/>